action.sh

```diff
@@ -26,38 +26,43 @@ echo 'Downloading hugo'
 curl -sSL https://github.com/spf13/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz > /tmp/hugo.tar.gz && tar -f /tmp/hugo.tar.gz -xz
 
 echo 'Building the hugo site'
-./hugo
+if [[ -z "$HUGO_ARGS" ]]; then
+    echo '$HUGO_ARGS not set'
+    echo 'Just running ./hugo'
+    ./hugo
+else
+    echo '$HUGO_ARGS set to'"$HUGO_ARGS"
+    ./hugo $HUGO_ARGS
```

Comment:
Do we need the if statement? If $HUGO_ARGS is empty, does it impact the command?

Reply:
Yes, if `$HUGO_ARGS` is empty, it won't effect the command. The if statement itself is a vestige of when it was originally for specifying theme (so a `-t` flag would be used for one, but not the other). I initially chose to leave it as is since it doesn't make that bit of a difference and makes the code a little bit more explicit. I can move the call to `hugo` outside the if statement if you'd like:

```bash
if [[ -z "$HUGO_ARGS" ]]; then
    echo '$HUGO_ARGS not set'
    echo 'Just running ./hugo'
else
    echo '$HUGO_ARGS set to'"$HUGO_ARGS"
fi
./hugo $HUGO_ARGS

Reply:
This looks better in my opinion. And maybe drop the "Just running ..." command, so you always get one statement around the HUGO_ARGS

What do you think?

Reply:
Sure, I've moved them into one statement. I kept the "just running" part because `$HUGO_ARGS not set` sounds almost like a warning/error.